vulcan/lib/orchestration.rb

```diff
@@ -220,16 +232,18 @@ def build_target_for(step_name, cache = {})
         elsif (step = workflow.find_step(step_name))
           if step.ui_query_name
             script = {}
+          elsif step.ui_output_name
+            script = {}
```

Comment:
@corps , I had to add cases for the `ui-outputs` directory for output scripts in `Orchestration`. Not sure if these were the right places to account for the new case, but it seems to work correctly?

Reply:
Yup, this works for me.  It ponders me if atleast on the backend we should have another method `ui_behavior?` which simply returns the boolean of wether ui_output_name or ui_query_name are set to make these conditionals simpler.